python/lsst/summit/utils/butlerUtils.py

```diff
@@ -150,6 +158,75 @@ def makeDefaultLatissButler(
     return butler
 
 
+def makeDefaultButler(
+    instrument: str,
+    *,
+    extraCollections: list[str] | None = None,
+    writeable: bool = False,
+    embargo: bool = True,
+) -> dafButler.Butler:
+    """Create a butler for the instrument using default collections, regardless
+    of the location.
+
+    Parameters
+    ----------
+    extraCollections : `list` of `str`
+        Extra input collections to supply to the butler init.
+    writable : `bool`, optional
+        Whether to make a writable butler.
+    embargo : `bool`, optional
+        Use the embargo repo instead of the main one. Needed to access
+        embargoed data if not at a summit-like location.
+
+    Returns
+    -------
+    butler : `lsst.daf.butler.Butler`
+        The butler.
+    """
+    SUPPORTED_SITES = [
+        "summit",
+        "tucson",
+        "base",
+        "staff-rsp",
+        "rubin-devl",
+        "usdf-k8s",
+    ]
+
+    site = getSite()
+    if site not in SUPPORTED_SITES:
+        raise ValueError(f"Default butler creation only supported at sites: {SUPPORTED_SITES}, got {site=}")
+
+    summitLike = site in ["summit", "tucson", "base"]
+    if summitLike:
+        if embargo is True:
+            logger = logging.getLogger(__name__)
+            logger.debug("embargo option is irrelevant on the summit, ignoring")
+            embargo = False  # there's only one repo too, so this makes the code more simple too
+
+    baseCollection = f"{instrument}/defaults"
+    raCollection = f"{instrument}/quickLook" if summitLike else f"{instrument}/nightlyValidation"
+
+    repo = instrument if embargo is False else "embargo"
+
+    collections = [baseCollection, raCollection]
+    if extraCollections is not None:
+        assert extraCollections is not None  # just for mypy
+        extraCollectionsList = ensure_iterable(extraCollections)
+        collections.extend(extraCollectionsList)
+
+    try:
+        butler = dafButler.Butler.from_config(
+            repo, collections=collections, writeable=writeable, instrument=instrument
+        )
+    except (FileNotFoundError, RuntimeError):
+        # Depending on the value of DAF_BUTLER_REPOSITORY_INDEX and whether
+        # it is present and blank, or just not set, both these exception
+        # types can be raised, see tests/test_butlerUtils.py:ButlerInitTestCase
+        # for details and tests which confirm these have not changed
+        raise FileNotFoundError  # unify exception type
```

Comment:
Do you want to include the original error message?

Reply:
This should be _incredibly_ rare tbh. I almost removed this who `try` really - this was put in in the very very early days before we reliably had the `$DAF_BUTLER_REPOSITORY_INDEX` set at all our sites, and sometimes it was set but pointed to a file that was missing, etc etc. I suspect this will simply never happen now, so I think this is OK.